packages/app-store/googlecalendar/lib/CalendarService.ts

```diff
@@ -372,6 +372,7 @@ export default class GoogleCalendarService implements Calendar {
                 name: cal.summary ?? "No name",
                 primary: cal.primary ?? false,
                 readOnly: !(cal.accessRole === "reader" || cal.accessRole === "owner") && true,
+                email: cal.id ?? "No Email",
```

Comment:
same here. do we wanna show "No Email"? or is this a non-visible error?

Reply:
didn't find a response where cal.id ( email) is not being passed. "No Email" is just to make sure email type is string just like  name: cal.summary ?? "No name",

Reply:
Could we render an empty string then not to cause confusion?

Reply:
ok . everywhere right?

Reply:
done